modules/peridynamics/doc/content/source/userobjects/GeneralizedPlaneStrainUserObjectHNOSPD.md

```diff
@@ -0,0 +1,11 @@
+# Self-stabilized Non-ordinary State-based Generalized Plane Strain UserObject
+
+## Description
+
+UserObject `GeneralizedPlaneStrainUserObjectHNOSPD` is used to provide the residual and Jacobian for the scalar out-of-plane strain field variable in a generalized plane strain formulation using the Self-stabilized Non-Ordinary State-based Peridynamic theory. The out-of-plane stress component is retrieved from the `Compute***Stress` Material class. Integral of this component over the whole simulation domain is accomplished by summation of product of material point area and the component value. Residual is the difference between stress integral and the applied force in the our-of-plane direction. Jacobian is the derivative of residual with respect to the out-of-plane strain.
```

Comment:
The ____ UserObject
Integral -> Integration
of product -> of the product
Residual -> The residual
Jacobian -> The Jacobian

Reply:
Did you want to say the the horizon stabilized formulation instead of self-stabilized?  Also, what's the story on the horizon stabilized formulation? Is that just a better name for the self-stabilized formulation, or is something different about it?

Reply:
I think horizon stabilized model is much better description of the method. Also, I will have a couple of more stabilized methods implemented later.